modules/battenberg/1.1/battenberg.smk

```diff
@@ -50,11 +50,48 @@ _battenberg_CFG = CFG
 
 # Define rules to be run locally when using a compute cluster
 localrules:
+    _battenberg_get_reference
     _battenberg_all
 
-
 ##### RULES #####
 
+# Downloads the reference files into the module results directory (under '00-inputs/') from https://www.bcgsc.ca/downloads/morinlab/reference/ . 
+rule _battenberg_get_reference:
```

Comment:
What will happen if a user provides genomes aligned to hs37d5? We need to handle all the genome builds covered by the reference workflow, not just the two we use in GAMBL.